scripts/broken-links-slack-payload.mjs

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-console */
```

Comment:
This is an adaptation of the script used for content release broken links. It's greatly simplified because we decided to abandon sending actual broken link information directly to Slack.